### DISCUSSION TITLE
/common-tool/hutool.html

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#28] [Announcements]
# /common-tool/hutool.html

一份通俗易懂、风趣幽默的Java学习指南，内容涵盖Java基础、Java并发编程、Java虚拟机、Java企业级开发、Java面试等核心知识点。学Java，就认准Java程序员进阶之路

https://tobebetterjavaer.com/common-tool/hutool.html

**Comment by wshidaijingshen** [comment]
棒棒哒
